tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1510,14 +1517,7 @@ def resize_image_with_pad_v2(image,
     If `images` was 4-D, a 4-D float Tensor of shape
     `[batch, new_height, new_width, channels]`.
     If `images` was 3-D, a 3-D float Tensor of shape
-    `[new_height, new_width, channels]`.
-	
-  Usage Example:
-    ```python
-    >> import tensorflow as tf
-    >> x = tf.random.normal(shape=(256, 256, 3))
-    >> tf.image.resize_with_pad(x, 200, 200, tf.image.ResizeMethod.BILINEAR, False)
-    ```
+    `[new_height, new_width, channels]`
```

Comment:
Please add the full stop as was in the original code.

`[new_height, new_width, channels]`.